docs/user/rstudio/ide/guide/productivity/custom-shortcuts.qmd

```diff
@@ -2,57 +2,61 @@
 title: "Custom Shortcuts"
 ---
 
-You can now customize keyboard shortcuts in the RStudio \-- you can bind keys to execute RStudio application commands, editor commands, or (using [RStudio Addins](https://rstudio.github.io/rstudioaddins/)) even user-defined R functions.
+Users can customize or bind keyboard shortcuts to execute: - RStudio application commands - Editor commands - User defined R functions or [RStudio Addins](https://rstudio.github.io/rstudioaddins/) or the [`shrtcts`](https://pkg.garrickadenbuie.com/shrtcts/) package
```

Comment:
This was supposed to be a list 😬 - it failed because of the lack of a return character or spacing after `to execute:`, which I've changed to the below:

```
Users can customize or bind keyboard shortcuts to execute:  

- RStudio application commands 
- Editor commands 
- User defined R functions 
- [RStudio Addins](https://rstudio.github.io/rstudioaddins/) or the [`shrtcts`](https://pkg.garrickadenbuie.com/shrtcts/) package
```